Core/include/Acts/Propagator/AtlasStepper.hpp

```diff
@@ -1416,7 +1418,7 @@ class AtlasStepper {
   std::shared_ptr<const MagneticFieldProvider> m_bField;
 
   /// Overstep limit: could/should be dynamic
-  double m_overstepLimit = -100 * UnitConstants::um;
+  double m_overstepLimit = 200 * UnitConstants::um;
```

Comment:
Numerical value probably doesn't need to change here?

Reply:
aligned this one to the `EigenStepper` one